### ISSUE TITLE
[question] Maybe a bug, maybe it's me...

### ISSUE THREAD

**Issue by tarnis** [alicevision/meshroom#849]
**Describe the problem**
I've been through three sets of images I took the other day, all of which are crapping out in the same general way.  My first test last week didn't generate a fantastic mesh but it at least made its way through.  

**Screenshots**

**Dataset**
https://photos.app.goo.gl/JcrsKpvK27KFZmNg8 

**Log**
`-rangeSize 40
 - logFile: C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/FeatureExtraction/da04ac47506bb5b11724cd73c4d3d756247a4424/2.log
 - elapsed time: 0:00:13.685010

[3/11] ImageMatching
 - commandLine: aliceVision_imageMatching  --input "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/CameraInit/76d25835d05cc346b94ab8262d80c561095453a5/cameraInit.sfm" --featuresFolders "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/FeatureExtraction/da04ac47506bb5b11724cd73c4d3d756247a4424" --tree "C:\Visual\Matts\Meshroom-2019.2.0\aliceVision\share\aliceVision\vlfeat_K80L3.SIFT.tree" --weights "" --minNbImages 200 --maxDescriptors 500 --nbMatches 50 --verboseLevel info --output "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/ImageMatching/6475fe9827714ccdf36a5001536b931905601ce7/imageMatches.txt"
 - logFile: C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/ImageMatching/6475fe9827714ccdf36a5001536b931905601ce7/log
 - elapsed time: 0:00:00.655250

[4/11](1/5) FeatureMatching
 - commandLine: aliceVision_featureMatching  --input "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/CameraInit/76d25835d05cc346b94ab8262d80c561095453a5/cameraInit.sfm" --featuresFolders "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/FeatureExtraction/da04ac47506bb5b11724cd73c4d3d756247a4424" --imagePairsList "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/ImageMatching/6475fe9827714ccdf36a5001536b931905601ce7/imageMatches.txt" --describerTypes sift --photometricMatchingMethod ANN_L2 --geometricEstimator acransac --geometricFilterType fundamental_matrix --distanceRatio 0.8 --maxIteration 2048 --geometricError 0.0 --maxMatches 0 --savePutativeMatches False --guidedMatching False --exportDebugFiles False --verboseLevel info --output "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/FeatureMatching/ef13c7bd87da649f4027c16e9208cfd8727bd570" --rangeStart 0 --rangeSize 20
 - logFile: C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/FeatureMatching/ef13c7bd87da649f4027c16e9208cfd8727bd570/0.log
 - elapsed time: 0:03:20.588895

[4/11](2/5) FeatureMatching
 - commandLine: aliceVision_featureMatching  --input "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/CameraInit/76d25835d05cc346b94ab8262d80c561095453a5/cameraInit.sfm" --featuresFolders "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/FeatureExtraction/da04ac47506bb5b11724cd73c4d3d756247a4424" --imagePairsList "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/ImageMatching/6475fe9827714ccdf36a5001536b931905601ce7/imageMatches.txt" --describerTypes sift --photometricMatchingMethod ANN_L2 --geometricEstimator acransac --geometricFilterType fundamental_matrix --distanceRatio 0.8 --maxIteration 2048 --geometricError 0.0 --maxMatches 0 --savePutativeMatches False --guidedMatching False --exportDebugFiles False --verboseLevel info --output "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/FeatureMatching/ef13c7bd87da649f4027c16e9208cfd8727bd570" --rangeStart 20 --rangeSize 20
 - logFile: C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/FeatureMatching/ef13c7bd87da649f4027c16e9208cfd8727bd570/1.log
 - elapsed time: 0:02:27.075351

[4/11](3/5) FeatureMatching
 - commandLine: aliceVision_featureMatching  --input "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/CameraInit/76d25835d05cc346b94ab8262d80c561095453a5/cameraInit.sfm" --featuresFolders "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/FeatureExtraction/da04ac47506bb5b11724cd73c4d3d756247a4424" --imagePairsList "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/ImageMatching/6475fe9827714ccdf36a5001536b931905601ce7/imageMatches.txt" --describerTypes sift --photometricMatchingMethod ANN_L2 --geometricEstimator acransac --geometricFilterType fundamental_matrix --distanceRatio 0.8 --maxIteration 2048 --geometricError 0.0 --maxMatches 0 --savePutativeMatches False --guidedMatching False --exportDebugFiles False --verboseLevel info --output "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/FeatureMatching/ef13c7bd87da649f4027c16e9208cfd8727bd570" --rangeStart 40 --rangeSize 20
 - logFile: C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/FeatureMatching/ef13c7bd87da649f4027c16e9208cfd8727bd570/2.log
 - elapsed time: 0:01:28.564709

[4/11](4/5) FeatureMatching
 - commandLine: aliceVision_featureMatching  --input "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/CameraInit/76d25835d05cc346b94ab8262d80c561095453a5/cameraInit.sfm" --featuresFolders "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/FeatureExtraction/da04ac47506bb5b11724cd73c4d3d756247a4424" --imagePairsList "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/ImageMatching/6475fe9827714ccdf36a5001536b931905601ce7/imageMatches.txt" --describerTypes sift --photometricMatchingMethod ANN_L2 --geometricEstimator acransac --geometricFilterType fundamental_matrix --distanceRatio 0.8 --maxIteration 2048 --geometricError 0.0 --maxMatches 0 --savePutativeMatches False --guidedMatching False --exportDebugFiles False --verboseLevel info --output "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/FeatureMatching/ef13c7bd87da649f4027c16e9208cfd8727bd570" --rangeStart 60 --rangeSize 20
 - logFile: C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/FeatureMatching/ef13c7bd87da649f4027c16e9208cfd8727bd570/3.log
 - elapsed time: 0:00:32.983792

[4/11](5/5) FeatureMatching
 - commandLine: aliceVision_featureMatching  --input "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/CameraInit/76d25835d05cc346b94ab8262d80c561095453a5/cameraInit.sfm" --featuresFolders "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/FeatureExtraction/da04ac47506bb5b11724cd73c4d3d756247a4424" --imagePairsList "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/ImageMatching/6475fe9827714ccdf36a5001536b931905601ce7/imageMatches.txt" --describerTypes sift --photometricMatchingMethod ANN_L2 --geometricEstimator acransac --geometricFilterType fundamental_matrix --distanceRatio 0.8 --maxIteration 2048 --geometricError 0.0 --maxMatches 0 --savePutativeMatches False --guidedMatching False --exportDebugFiles False --verboseLevel info --output "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/FeatureMatching/ef13c7bd87da649f4027c16e9208cfd8727bd570" --rangeStart 80 --rangeSize 20
 - logFile: C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/FeatureMatching/ef13c7bd87da649f4027c16e9208cfd8727bd570/4.log
 - elapsed time: 0:00:00.549532

[5/11] StructureFromMotion
 - commandLine: aliceVision_incrementalSfM  --input "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/CameraInit/76d25835d05cc346b94ab8262d80c561095453a5/cameraInit.sfm" --featuresFolders "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/FeatureExtraction/da04ac47506bb5b11724cd73c4d3d756247a4424" --matchesFolders "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/FeatureMatching/ef13c7bd87da649f4027c16e9208cfd8727bd570" --describerTypes sift --localizerEstimator acransac --localizerEstimatorMaxIterations 4096 --localizerEstimatorError 0.0 --lockScenePreviouslyReconstructed False --useLocalBA True --localBAGraphDistance 1 --maxNumberOfMatches 0 --minInputTrackLength 2 --minNumberOfObservationsForTriangulation 2 --minAngleForTriangulation 3.0 --minAngleForLandmark 2.0 --maxReprojectionError 4.0 --minAngleInitialPair 5.0 --maxAngleInitialPair 40.0 --useOnlyMatchesFromInputFolder False --useRigConstraint True --lockAllIntrinsics False --initialPairA "" --initialPairB "" --interFileExtension .abc --verboseLevel info --output "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/StructureFromMotion/e6ee24befbf44e5efbff09d2a82a5a6e0e780657/sfm.abc" --outputViewsAndPoses "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/StructureFromMotion/e6ee24befbf44e5efbff09d2a82a5a6e0e780657/cameras.sfm" --extraInfoFolder "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/StructureFromMotion/e6ee24befbf44e5efbff09d2a82a5a6e0e780657"
 - logFile: C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/StructureFromMotion/e6ee24befbf44e5efbff09d2a82a5a6e0e780657/log
 - elapsed time: 0:00:07.005778
ERROR:root:Error on node computation: Error on node "StructureFromMotion_1":
Log:
Program called with the following parameters:
 * describerTypes = "sift"
 * extraInfoFolder = "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/StructureFromMotion/e6ee24befbf44e5efbff09d2a82a5a6e0e780657"
 * featuresFolders =  = [C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/FeatureExtraction/da04ac47506bb5b11724cd73c4d3d756247a4424]
 * initialPairA = ""
 * initialPairB = ""
 * input = "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/CameraInit/76d25835d05cc346b94ab8262d80c561095453a5/cameraInit.sfm"
 * interFileExtension = ".abc"
 * localBAGraphDistance = 1
 * localizerEstimator =  Unknown Type "enum aliceVision::robustEstimation::ERobustEstimator"
 * localizerEstimatorError = 0
 * localizerEstimatorMaxIterations = 4096
 * lockAllIntrinsics = 0
 * lockScenePreviouslyReconstructed = 0
 * matchesFolders =  = [C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/FeatureMatching/ef13c7bd87da649f4027c16e9208cfd8727bd570]
 * maxAngleInitialPair = 40
 * maxNumberOfMatches = 0
 * maxReprojectionError = 4
 * minAngleForLandmark = 2
 * minAngleForTriangulation = 3
 * minAngleInitialPair = 5
 * minInputTrackLength = 2
 * minNumberOfObservationsForTriangulation = 2
 * output = "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/StructureFromMotion/e6ee24befbf44e5efbff09d2a82a5a6e0e780657/sfm.abc"
 * outputViewsAndPoses = "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/StructureFromMotion/e6ee24befbf44e5efbff09d2a82a5a6e0e780657/cameras.sfm"
 * useLocalBA = 1
 * useOnlyMatchesFromInputFolder = 0
 * useRigConstraint = 1
 * useTrackFiltering = 1 (default)
 * verboseLevel = "info"

Loading features
0%   10   20   30   40   50   60   70   80   90   100%
|----|----|----|----|----|----|----|----|----|----|
[08:12:42.205891][info] Supported CUDA-Enabled GPU detected.

WARNING: downgrade status on node "PrepareDenseScene_1(0)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "PrepareDenseScene_1(1)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "PrepareDenseScene_1(2)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMap_1(0)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMap_1(1)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMap_1(2)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMap_1(3)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMap_1(4)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMap_1(5)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMap_1(6)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMap_1(7)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMap_1(8)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMap_1(9)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMap_1(10)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMap_1(11)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMap_1(12)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMap_1(13)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMap_1(14)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMap_1(15)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMap_1(16)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMap_1(17)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMap_1(18)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMap_1(19)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMap_1(20)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMap_1(21)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMap_1(22)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMap_1(23)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMap_1(24)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMap_1(25)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMap_1(26)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMapFilter_1(0)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMapFilter_1(1)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMapFilter_1(2)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMapFilter_1(3)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMapFilter_1(4)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMapFilter_1(5)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMapFilter_1(6)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMapFilter_1(7)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "DepthMapFilter_1(8)" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "Meshing_1" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "MeshFiltering_1" from Status.SUBMITTED to Status.NONE
WARNING: downgrade status on node "Texturing_1" from Status.SUBMITTED to Status.NONE
ERROR:root:Error during Graph execution Error on node "StructureFromMotion_1":
Log:
Program called with the following parameters:
 * describerTypes = "sift"
 * extraInfoFolder = "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/StructureFromMotion/e6ee24befbf44e5efbff09d2a82a5a6e0e780657"
 * featuresFolders =  = [C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/FeatureExtraction/da04ac47506bb5b11724cd73c4d3d756247a4424]
 * initialPairA = ""
 * initialPairB = ""
 * input = "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/CameraInit/76d25835d05cc346b94ab8262d80c561095453a5/cameraInit.sfm"
 * interFileExtension = ".abc"
 * localBAGraphDistance = 1
 * localizerEstimator =  Unknown Type "enum aliceVision::robustEstimation::ERobustEstimator"
 * localizerEstimatorError = 0
 * localizerEstimatorMaxIterations = 4096
 * lockAllIntrinsics = 0
 * lockScenePreviouslyReconstructed = 0
 * matchesFolders =  = [C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/FeatureMatching/ef13c7bd87da649f4027c16e9208cfd8727bd570]
 * maxAngleInitialPair = 40
 * maxNumberOfMatches = 0
 * maxReprojectionError = 4
 * minAngleForLandmark = 2
 * minAngleForTriangulation = 3
 * minAngleInitialPair = 5
 * minInputTrackLength = 2
 * minNumberOfObservationsForTriangulation = 2
 * output = "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/StructureFromMotion/e6ee24befbf44e5efbff09d2a82a5a6e0e780657/sfm.abc"
 * outputViewsAndPoses = "C:/Visual/Matts/Meshroom-2019.2.0/MeshroomCache/StructureFromMotion/e6ee24befbf44e5efbff09d2a82a5a6e0e780657/cameras.sfm"
 * useLocalBA = 1
 * useOnlyMatchesFromInputFolder = 0
 * useRigConstraint = 1
 * useTrackFiltering = 1 (default)
 * verboseLevel = "info"

Loading features
0%   10   20   30   40   50   60   70   80   90   100%
|----|----|----|----|----|----|----|----|----|----|
[08:12:42.205891][info] Supported CUDA-Enabled GPU detected.

`
**Desktop (please complete the following and other pertinent information):**
 - OS: Windows 10 64 bit
 - Python version 3.6.6
 - Qt/PySide version Dunno
 - Meshroom version: Release
   - Binary version 2019.2.0

**Additional context**

**Comment by natowi** [comment]
Your images do not have enough overlap (60-70% is recommended). Many images are too dark and your tree stump does not fill most of the image frame.

**Comment by tarnis** [comment]
Something else needs to be going on here...I can upload that complete image set if you like, that was a couple of them as an example.

I have tried a few more, all fail.  My original one that worked of a scan of my kitchen now fails as well.  I originally had it on my desktop when I ran it and then moved it to another folder if that might have something to do with things?  Reg key somewhere?

I downloaded the doll from this page which also fails in the same way:
https://www.agisoft.com/downloads/sample-data/
